tools/src/test/python/dlpx/virtualization/_internal/fake_plugin/direct/successful.py

```diff
@@ -45,12 +47,12 @@ def mount_specification(repository, virtual_source):
 
 
 @direct.virtual.post_snapshot()
-def postSnapshot(repository, source_config, virtual_source):
+def post_snapshot(repository, source_config, virtual_source):
     return None
 
 
 @direct.virtual.pre_snapshot()
-def preSnapshot(repository, source_config, virtual_source):
+def pre_snapshot(repository, source_config, virtual_source):
```

Comment:
Should we add `snapshot_parameters` to these pre/post snapshot functions too?

Reply:
nah this is virtual not linked. I was just fixing the syntax because I noticed they were wrong.